sdk/cognitivelanguage/azure-ai-language-conversations/swagger/README.md

```diff
@@ -149,10 +158,50 @@ directive:
         $["parameters"][1]["x-ms-client-name"] = "task";
 ```
 
+```yaml $(tag) == 'release_runtime_1_1_preview'
+# Rename begin_conversation_analysis `body` to `tasks`
+directive:
+    - from: swagger-document
+      where: $["paths"]["/analyze-conversations/jobs"]["post"]
+      transform: >
+        $["parameters"][1]["x-ms-client-name"] = "task";
+```
+
+```yaml $(tag) == 'release_runtime_1_1_preview'
+# Give analyze job LRO a return type
+directive:
+  - from: swagger-document
+    where: '$.paths["/analyze-conversations/jobs"].post'
+    transform: >
+      $["responses"]["200"] = {
+          "description": "dummy schema to get poller response when calling .result()",
+          "schema": {
+              "$ref": "#/definitions/AnalyzeConversationJobState"
+          }
+      };
+```
+
+```yaml $(tag) == 'release_runtime_1_1_preview'
+# Remove async GET operation status
+directive:
+    - from: swagger-document
+      where: $["paths"]
+      transform: >
+          delete $["/analyze-conversations/jobs/{jobId}"];
+```
+
+```yaml $(tag) == 'release_runtime_1_1_preview'
+# Remove async cancel operation
```

Comment:
Interesting. Why? Do you not support cancel, or is it built-in? That said, it shouldn't return an object like I did (and maybe you copied) for the inference client, but doesn't hurt. Just not a pattern we (the SDK teams) want to do going forward after talking with Johan et. al.

Reply:
I posted in our chat yesterday that we're not ready to expose this quite yet:

> I spoke to Johan about cancel and he would like us to skip it for this release. DPG currently does not provide the poller metadata to cancel an operation and is something they will work on. Therefore the 1.1.0b2 for Python will only add the authoring APIs and regen with GA DPG.